packaging/installer/netdata-updater.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/sh
```

Comment:
Please, change for `/bin/bash`, because `sh` is not symbolic link for `/bin/bash` on all platforms.

Reply:
The use of `/bin/sh` is intentional, see the above discussion with Vasilis.